docs/remote/containers.md

```diff
@@ -73,34 +69,18 @@ To get started, follow these steps:
 
 **Next Steps:** The Remote - Containers extension supports two primary operating models:
```

Comment:
I was looking for "Next Steps" at the bottom of the page and wasn't anticipating to find them here. While it does seem to fit into the flow here, should we also add it somewhere to the bottom of the doc?